steps/bitrise-step-xcode-project-info/2.1.0/step.yml

```diff
@@ -0,0 +1,58 @@
+title: Xcode Project Info
+summary: Extracts Xcode project info data to environment variables.
+description: |-
+  This step will simply extract Xcode project information (Version and Build),
+  and export them into environment variables prefixed with **XPI_**.
+
+  Afterwards, you can use these environment variables in other steps (ie. sending message on Slack).
+website: https://github.com/tadija/bitrise-step-xcode-project-info
+source_code_url: https://github.com/tadija/bitrise-step-xcode-project-info
+support_url: https://github.com/tadija/bitrise-step-xcode-project-info/issues
+published_at: 2020-04-17T18:23:26.439085+02:00
+source:
+  git: https://github.com/tadija/bitrise-step-xcode-project-info.git
+  commit: afd75d12e414fbdb3907a0a5cde5b79aa299a65e
+host_os_tags:
+- osx-10.10
+project_type_tags:
+- ios
+type_tags:
+- xcode
```

Comment:
Please check the contribution guide for valid type_tags:
https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md

> Available type_tags:
>
>access-control
>artifact-info
>installer
>deploy
>utility
>dependency
>code-sign
>build
>test
>notification